docs/DeveloperGuide.md

```diff
@@ -691,22 +691,81 @@ testers are expected to do more *exploratory* testing.
+   
+   
+   2. Test valid input values
+      - E.G. `edit INDEX [Optional parameters]` <br>
+        Test input: `edit 1 n/new name` when list has more than 1 application <br>
+        Expected output: Successful change of application 1 name
+   3. Test invalid input values
+      - E.G.  `edit INDEX [Optional parameters]` <br>
+        Test input: `edit 0 n/new name` or `edit -10 n/new name` <br>
+        Expected output: error message detailing what went wrong
+   4. Test invalid command input
+       - E.G.  `edit INDEX [Optional parameters]` <br>
+         Test input: `edi 1 n/new name` or `edit 2 2 n/new name` <br>
+         Expected output: error message detailing what went wrong
+   5. Test multiple parameter combinations
+       - E.G. `edit INDEX [optional parameters]` <br>
+         Test input: `edit 1 n/new name n/actual name` <br>
+         Expected output: error message detailing what went wrong
+         Actual output: command executes
+
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** Whenever unsure if a behaviour is intended or not,
+consult the [User Guide](https://ay2122s2-cs2103t-t11-3.github.io/tp/UserGuide.html) first. If not documented there, feel free to raise the issue.
 
-1. Deleting a person while all persons are being shown
+</div>
+### Example: Deleting an application
+
+1. Deleting an application while all applications are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: List all applications using the `list` command. Multiple applications in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First application is deleted from the list. Details of the deleted application shown in the status message.
 
    1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No application is deleted. Error details shown in the status message. Status bar remains the same.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+2. Deleting an application while reminder window is open
+   1. Prerequisites: Have the reminder window open using `reminder` and 
+   set the first applications interview date to be within the next 7 days.
+
+   2. Test case: `delete 1`<br>
+      Expected: First application is deleted from the list. Details of the deleted application shown in the status message.
+      Application is also deleted from reminder window
+
+   3. Test case: `delete 0`<br>
+      Expected: No application is deleted. Error details shown in the status message. Status bar remains the same.
```

Comment:
Did you mean the summary bar remains the same?

Reply:
I was following the previous examples, where they referenced a status bar, assuming it refers to the IDE status bar